lib/face.js

```diff
@@ -0,0 +1,42 @@
+const Point = require('./point');
+
+class Face{
+  constructor(pointWiseEdges, loc, color = [100,100,100]){
+    this.edges = pointWiseEdges;
+    this.loc = loc;
+    this.color = color;
+  }
+
+  xyzEndpointsRelativeTo(player){
+    return this.edges.map(function(edge){
+      var s_t = edge.xyzEndpointsRelativeTo(player);
```

Comment:
Maybe be more explicit with the variable names? Also, I laugh at your use of underscores. :sparkling_heart: